src/Game.js

```diff
@@ -0,0 +1,93 @@
+function Game() {
+  this._frames = [];
+  this._currentFrameNumber = 1;
+  this._numberOfFrames = 10;
+  this._frames.push(new Frame(false))
+
+<<<<<<< HEAD
```

Comment:
Similar comment to above. This line indicates that a merge conflict was not properly resolved.